fix: duplicated entry (#34458)
mdn/content: files/en-us/glossary/sld/index.md

{{GlossarySidebar}}

> A Second Level Domain (SLD) is the part of the domain name that is located right before a Top Level Domain ({{Glossary("TLD")}}).

Reviewer: ```suggestion
The **Second Level Domain (SLD)** is the part of the domain name that is located right before a {{Glossary("TLD", "Top Level Domain (TLD)")}}_.
```

For example, in `mozilla.org` the SLD is `mozilla` and the TLD is `org`.

> A domain name is not limited to a TLD and an SLD. Additional subdomains can be created in order to provide additional information about various functions of a server or to delimit areas under the same domain. For example, `www` is a commonly used subdomain to indicate the domain points to a web server.

Reviewer: ```suggestion
A domain name is not limited to a TLD and SLD. Additional subdomains can be created to provide additional information about various server functions or to delimit areas under the same domain. For example, `www` is a commonly used subdomain to indicate that the domain points to a web server.
```
